controllers/imageclusterinstall_monitor.go

```diff
@@ -93,6 +93,14 @@ func (r *ImageClusterInstallMonitor) monitorInstallationProgress(
 	}
 	if !bmh.Status.PoweredOn {
 		log.Infof("BareMetalHost %s/%s is not powered on yet", bmh.Name, bmh.Namespace)
+		timedout, err := r.handleClusterTimeout(ctx, log, ici, r.DefaultInstallTimeout, "BMH failed to power on within the cluster installation timeout (%s)")
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+		if timedout {
+			// in case of timeout we want to requeue after 1 hour
+			return ctrl.Result{RequeueAfter: time.Hour}, nil
```

Comment:
@eranco74 we want to requeu after an hour? I don't see it in ticket, as for it should be fine

Reply:
In case of a timeout during the installation, we [requeue after an hour](https://github.com/openshift/image-based-install-operator/blob/main/controllers/imageclusterinstall_monitor.go#L150-L153). I kept it an hour here as well because I didn't see why there should be a different between the 2 cases.

Reply:
I see, as we are not watching bmh in this controller, seems logical to do it. Thanks

Reply:
Should we maybe log here? At least debug?

Reply:
I think we can move the code added here outside of the `if !bmh.Status.PoweredOn` scope (move it to line 94).
The timeout condition message doesn't need to specify the BMH condition (The BMH status will be reflected in the installing condition message) .
This should allow removing the call to`handleClusterTimeout` from `checkClusterStatus` .
Thoughts?

Reply:
NM, if we do that we can't recover from timeout.
So just add the log (info is fine).
Note that the the timeout condition message can be the same regardless of the BMH status

Reply:
I added the log and reverted my changes to the message so it will be the same